### PR TITLE
feature github apps

### PR DESCRIPTION
**Pull request by Ldcabansay**
features dockstore github app links to [Dockstore GitHub App](https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html) landing page
image

### CODE REVIEW

**Review by garyluu — COMMENTED** [review]
https://github.com/dockstore/dockstore/issues/3407

File: feat-content.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="https://bcc2020.github.io/">
-            <h4 class="feat-content-title">Bioinformatics Community Conference 2020</h4>
-            <p class="feat-card-description">Registration for BCC2020 is now open! Check out the full program and sign up for Dockstore's virtual workshops on Docker, Descriptors, and analysis with Terra.</p>
+           href="https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html">
+            <h4 class="feat-content-title">Dockstore GitHub App for Continuous Integration</h4>
+            <p class="feat-card-description">Our new Dockstore GitHub App adds support for registering and automatically syncing workflows and services between GitHub and Dockstore. Checkout our guides for more details.</p>
```
Comment (coverbeck) [inline feat-content.html:7]: As a verb phrase, check out is two words.

File: feat-content.html [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="https://bcc2020.github.io/">
-            <h4 class="feat-content-title">Bioinformatics Community Conference 2020</h4>
-            <p class="feat-card-description">Registration for BCC2020 is now open! Check out the full program and sign up for Dockstore's virtual workshops on Docker, Descriptors, and analysis with Terra.</p>
+           href="https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html">
+            <h4 class="feat-content-title">Dockstore GitHub App for Continuous Integration</h4>
```
Comment (coverbeck) [inline feat-content.html:6]: I don't think I like "Continuous Integration" -- for me it implies a build server. I would just say "Dockstore GitHub App".

**Review by coverbeck — APPROVED** [review]
My opinion on Continuous Integration is subjective and I'm willing to concede that one.

Feel more strongly "check out".

**Review by garyluu — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Pull request merged**